partitioned-table.md

```diff
@@ -987,43 +991,92 @@ When reorganizing partitions, you need to note the following key points:
++---------+------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
 
-Currently, `ALTER TABLE ... COALESCE PARTITION` is not supported in TiDB as well. For partition management statements that are not currently supported, TiDB returns an error.
+### Manage Hash and Key partitions
 
-{{< copyable "sql" >}}
+This section uses the partitioned tables created by the following SQL statements as examples to show you how to manage Hash and Key partitions.
 
 ```sql
-ALTER TABLE MEMBERS OPTIMIZE PARTITION p0;
+CREATE TABLE example (id int primary key, data varchar(1024)) PARTITION BY HASH(id) PARTITIONS 2;
 ```
 
+Increase the number of partitions by 1:
+
 ```sql
-ERROR 8200 (HY000): Unsupported optimize partition
+ALTER TABLE example ADD PARTITION PARTITIONS 1;
 ```
 
-### Key partition management
+You can also specify partition options by adding partition definitions:
 
-Currently, Key partitioning only supports the `ALTER TABLE ... TRUNCATE PARTITION` partition management statement.
+```sql
+ALTER TABLE example ADD PARTITION
+(PARTITION pExample4 COMMENT = 'not p3, but pExample4 instead',
+ PARTITION pExample5 COMMENT = 'not p4, but pExample5 instead');
+```
+
+Unlike Range and List partitioning, `DROP PARTITION` is not supported for Hash and Key partitioning, but you can decrease the number of partitions with `COALESCE PARTITION` or delete all data from specific partitions with `TRUNCATE PARTITION`
+
+Decreasing number of partitions by 1:
 
 ```sql
-ALTER TABLE members TRUNCATE PARTITION p0;
+ALTER TABLE example COALESCE PARTITION 1;
 ```
 
+> **Notes:**
+> 
+> - Changing the number of partitions for Hash or Key partitioned tables will do an online reorganization and copy all data to the new number of partitions.
+> - After changing the number of partitions, the statistics need to be updated.
+
+```sql
++---------+------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Level   | Code | Message                                                                                                                                                |
++---------+------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Warning | 1105 | The statistics of related partitions will be outdated after reorganizing partitions. Please use 'ANALYZE TABLE' statement if you want to update it now |
++---------+------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+1 row in set (0.00 sec)
 ```
-Query OK, 0 rows affected (0.03 sec)
+
+Result:
+```sql
+SHOW CREATE TABLE\G
```

Comment:
Do we need to update the statememt to `SHOW CREATE TABLE example;`

Reply:
I think it is still correct.